app/screens/settings/report_problem/report_problem.test.tsx

```
@@ -4,10 +4,12 @@
 import {act, fireEvent} from '@testing-library/react-native';
 import React from 'react';
 
+import {DEFAULT_REPORT_A_PROBLEM_EMAIL} from '@constants/report_a_problem';
 import {Screens} from '@constants';
```

ESLint

[error] `@constants` import should occur before import of `@constants/report_a_problem`

(import/order)